Alvaro's AD review comments (#47)
ietf-gitwg/using-github: draft-ietf-git-using-github.md

   email: barbara.stark@att.com

informative:
  GLOSSARY:
    title: "GitHub glossary"
    date: 2020-03

>     target: "https://help.github.com/en/github/getting-started-with-github/github-glossary"

Reviewer: Are we concerned about this link not aging well? (I wonder if we should, in parallel, try to move the contents of https://ietf-gitwg.github.io to some place more stable and under control of the IETF.)

Author: Yeah, but I'm not sure that our own copy of this is guaranteed to be any better.  The GitHub link is nice in the sense that it concentrates on terminology.

Reviewer: Yep, no argument there. It certainly has better content.

As documents mature further, explicit confirmation of technical decisions with
the Working Group mailing list becomes more important.

> Chairs are able to declare Working Group consensus about the resolution of

Reviewer: ```suggestion
Chairs can declare Working Group consensus about the resolution of
```
